src/auth/auth.controller.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { AuthService } from 'src/auth/auth.service';
+import { Controller, Post, Body, Res, Req } from '@nestjs/common';
+import { LoginDto } from 'src/auth/dto/login-user.dto';
+import { Request, Response } from 'express';
+import { RegisterDto } from 'src/auth/dto/register-user.dto';
+
+@Controller('/api')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('/login')
+  async login(
+    @Req() request: Request,
+    @Res() response: Response,
+    @Body() loginDto: LoginDto,
+  ): Promise<any> {
+    try {
+      const result = await this.authService.login(loginDto);
+      return response.status(200).json({
```

Comment:
```suggestion
      return response.status(<...>).json({
```
em dùng enum HttpStatus của package @nestjs/common nhé,
ko hardcode `200` trực tiếp như kia